### PR TITLE
added navbar and logout with local storage

### PR DESCRIPTION
**Pull request by damfinkel**
## Summary

Navbar and logout

## Screenshots

pivot

### CODE REVIEW

File: src/components/NavBar.vue [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<template lang="pug">
+  nav.navbar-container
+    img.navbar-image(src="../assets/wolox-navbar-icon.png")
+    button.logout-button(type="button" @click="logout") Logout
```
Comment (mcallegari10) [inline src/components/NavBar.vue:4]: We write the plain text with a pipe notation

```pug
button
  |  Logout
```

File: src/components/NavBar.vue [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+</template>
+
+<style scoped lang="scss">
+@import '../scss/application';
```
Comment (mcallegari10) [inline src/components/NavBar.vue:8]: As I said in #1 you should only import this in the main component

File: src/components/NavBar.vue [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  max-height: 70%;
+}
+
+.logout-button {
+  color: black;
+  font-size: 18px;
+}
+</style>
+
+<script>
```
Comment (mcallegari10) [inline src/components/NavBar.vue:31]: We usually order like: `template` - `script` - `styles`

File: src/router.js [hunk 4]
```diff
@@ -16,8 +17,8 @@ export default new Router({
       component: Login
     },
     {
-      path: '/',
-      redirect: '/login'
+      path: '/home',
+      component: Home
```
Comment (mcallegari10) [inline src/router.js:21]: Remember to use lazy loading

**Review by mcallegari10 — APPROVED** [review]

File: src/components/NavBar.vue (hunk 2 above)
Comment (damfinkel) [reply src/components/NavBar.vue:8]: Fixed in another PR

File: src/components/NavBar.vue (hunk 3 above)
Comment (damfinkel) [reply src/components/NavBar.vue:31]: Will fix in a future PR to avoid possible conflicts, started doing it that way since 3 PRs ago aprox

**Pull request merged**